cluster/gce/util.sh

```diff
@@ -253,6 +253,7 @@ function copy-to-staging() {
     fi
   fi
 
+  rm -f "${tar}.sha512"
```

Comment:
we shouldn't need to do this? `>` below should already overwrite, `>>` would be append? what am I missing?

Reply:
I ran into an issue when running the e2e tests locally. If the file already exists from a previous run, it is sometimes created without write permissions. So the following command fails: `echo "${hash}" > "${tar}.sha512"` . I think it had to do with my `umask`. I can remove this if you want.

Reply:
ooh, we should defend against umask issues when manipulating tarballs etc.

I'm not sure what the best answer is here. in some other places where we're containerized we've explicitly set the permission bits.

TODO: follow-up on umask fun and these scripts